_events/2023/2023-07-community-call.md

```diff
@@ -0,0 +1,19 @@
+---
+title: July 2023 Community Call
+expires: 2023-07-14
+event_date: "July 13, 2023"
+layout: event
+duration: 60
+repeated: false
+category: community-call
+time:
+    - - start: 2023-07-13T16:00:00Z
+        end: 2023-07-13T17:00:00Z
+---
+
+Come join us for the July 2023 Community Call - **A Retrospective on Pride Month: Celebrating Diversity in Our Community!**  The Diversity, Equity, and Inclusion (DEI) Working Group will be leading the discussion on how the community has celebrated Pride Month this year, ways everyone in the community can engage with LGBTQ+ diversity throughout the year, and have an open discussion about how to continue these efforts in the future for Pride Month and other diverse communities.  As a change of pace to keep things fun, you will have the opportunity to select the breakout room on the topic that most interests you.  As a primer to help guide the discussion, feel free to contemplate some questions about allyship here.
```

Comment:
I think the link for the allyship doc is missing.

Reply:
Good catch--thanks! I added it.